### ISSUE TITLE
Latest core from WinUAE

### ISSUE THREAD

**Issue by mankeli** [FrodeSolheim/fs-uae#331]
Hi, would it possible to get the latest WinUAE core to fs-uae? There has been quite a lot of improvements in there (the current one seems to be from 2017?) and i'm again hitting a bug in blitter implementation. :)

**Comment by ztronzo** [comment]
To my knowledge Frode replied to the same question in the past that FS-UAE 3.x will remain with WinUAE core 3.30b and FS-UAE 4.x is using WinUAE core 4.2.1

I know some fixes were applied since WinUAE 4.1.0 which do fix some annoying visual/blitter issues

**Comment by emoon** [comment]
Friendly poke on this one. As WinUAE 5.0 has been released I wonder if/when the latest code will be merged into fs-uae?

**Comment by fuzzy76** [comment]
No commits or support forum replies for a year. I fear it's dead. :-/

**Comment by intrepidsilence** [comment]
Yes, could we get an update at least as to whether FS-UAE is dead? Good alternatives? Amiberry?

**Comment by emoon** [comment]
@intrepidsilence I would recommend Amiberry yes
